### PR TITLE
End Screen Implementation

### PR DESCRIPTION
**Pull request by FaeBurns**

### CODE REVIEW

**Comment by FaeBurns** [comment]
Don't merge until fix/obj-spawn has been both merged into main and into this. The OnArrest Sus NPC filter will need to be reworked to fit the new item spawning system

**Review by SolarBAM404 — APPROVED** [review]

**Pull request merged**
